lib/OpenQA/Schema/Result/Jobs.pm

```diff
@@ -1040,7 +1035,6 @@ sub update_backend ($self, $backend_info) {
     $self->update(
         {
             backend => $backend_info->{backend},
-            backend_info => encode_json($backend_info->{backend_info})});
```

Comment:
You still need the closing braces and semicolon.

Reply:
Sorry, with stupid mistakes like those I really feel I am not yet 100% healthy again